packages/framework/src/Model/Customer/Exception/EmptyCustomerUserIdentifierException.php

```diff
@@ -4,7 +4,7 @@
 
 use Exception;
 
-class EmptyCustomerIdentifierException extends Exception implements CustomerException
+class EmptyCustomerUserIdentifierException extends Exception implements CustomerUserException
```

Comment:
I think, that this changes should be committed in previous commit "renamed User* objects to CustomerUser*"
Maybe we can squash these two commits